src/odemis/driver/smaract.py

```diff
@@ -1240,6 +1240,7 @@ def __init__(self, name, role, locator, axes, ref_on_init=False, linear_speed=0.
         # VA dict str(axis) -> bool
         self.referenced = model.VigilantAttribute(axes_ref, readonly=True)
         # If ref_on_init, referenced immediately.
+        self._deactive_pos_after_ref = None
```

Comment:
Please, don't make it automatic. There should still be an argument in `__init__` called something like `pos_deactive_after_ref` to indicate the user wants this behaviour. 
That's needed for both drivers.